### PR TITLE
add ros-xmlrpcpp

### PR DESCRIPTION
**Pull request by wolfv**
Checklist

- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ ] Source is from official source
- [ ] Package does not vend other packages
- [ ] Build number is 0
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/xmlrpcpp/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
+package:
+  name: ros-xmlrpcpp
+  version: 1.14.3
+
+source:
+  url: https://github.com/ros-gbp/ros_comm-release/archive/release/melodic/xmlrpcpp/1.14.3-0.tar.gz
+  fn: ros-xmlrpcpp-1.14.3.tar.gz
+  sha256: d141854dd4371212f6606bfff6f7a44eb1a0292bbcf64cf29f4fed0c0741bc76
+  patches:
```
Comment (xhochy) [inline recipes/xmlrpcpp/meta.yaml]:
```suggestion
```

File: recipes/xmlrpcpp/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+test:
+  requires:
+    - rospkg
+  commands:
+    - export ROS_ROOT=$PREFIX && python -c "import rospkg;s = rospkg.RosPack();ret = s.get_path('xmlrpcpp');print(ret)"
+
+about:
+  home: http://xmlrpcpp.sourceforge.net
+  summary: |
+    XmlRpc++ is a C++ implementation of the XML-RPC protocol. This version is heavily modified from the
```
Comment (xhochy) [inline recipes/xmlrpcpp/meta.yaml]: Open ended sentence

**Comment by wolfv** [comment]
this one also skips windows cc @xhochy do you want me to fix the comments or is it fine?

**Comment by xhochy** [comment]
Fine with the fixed scripts.

**Pull request merged**